presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -283,6 +284,11 @@ public SystemSessionProperties(
                         featuresConfig.isLegacyOrderBy(),
                         false),
                 booleanSessionProperty(
+                        SUPPORT_NULL_ON_LHS_OF_SEMI_JOIN,
```

Comment:
@martint as i promised i run tpcds without and with this change. For the queries containing `IN` (15 of them, including `qNN_N` variants) no performance regression was observed.

Still, maybe we want the switch temporarily, to be on the safe side?

@KBP-TDC any thought?